### PR TITLE
Cato 4027 fixing errors

### PR DESCRIPTION
**Pull request by mario-paniccia**

### CODE REVIEW

File: src/test/scala/uk/gov/hmrc/ct/accounts/frs10x/AccountsFreeTextValidationFixture.scala [hunk 1]
```diff
@@ -65,7 +65,7 @@ trait AccountsFreeTextValidationFixture extends WordSpec with Matchers with Mock
   def testAccountsCoHoTextFieldValidation(boxId: String, builder: (Option[String]) => ValidatableBox[AbridgedAccountsBoxRetriever]): Unit = {
     setUpMocks()
     "fail validation if invalid characters" in {
-      builder(Some("^ §")).validate(boxRetriever) shouldBe Set(CtValidation(Some(boxId), s"error.$boxId.regexFailure", Some(List("^, §"))))
+      builder(Some("^ §")).validate(boxRetriever) shouldBe Set(CtValidation(Some(boxId), s"error.$boxId.regexFailure", Some(List("^  §"))))
```
Comment (tmikus) [inline src/test/scala/uk/gov/hmrc/ct/accounts/frs10x/AccountsFreeTextValidationFixture.scala]: Is this space necessary?

File: src/test/scala/uk/gov/hmrc/ct/box/ValidatableBoxSpec.scala [hunk 2]
```diff
@@ -188,7 +188,7 @@ class ValidatableBoxSpec  extends WordSpec with MockitoSugar  with Matchers with
 
   "validateCoHoOptionalTextField" should {
     "return unique errors if it does not match" in {
-      validateCoHoOptionalString("testBox", testOptStringBox(Some("^ ^^  aa § 333"))) shouldBe Set(CtValidation(Some("testBox"), "error.testBox.regexFailure", Some(List("^, §"))))
+      validateCoHoOptionalString("testBox", testOptStringBox(Some("^ ^^  aa § 333"))) shouldBe Set(CtValidation(Some("testBox"), "error.testBox.regexFailure", Some(List("^  §"))))
```
Comment (tmikus) [inline src/test/scala/uk/gov/hmrc/ct/box/ValidatableBoxSpec.scala]: Same here - is this space necessary?

File: src/main/scala/uk/gov/hmrc/ct/box/ValidatableBox.scala [hunk 3]
```diff
@@ -257,7 +257,7 @@ trait ValidatableBox[T <: BoxRetriever] extends Validators {
       val p = Pattern.compile(ValidCoHoCharacters)
       val m = p.matcher(x)
       val allMatchedCharsPluses = m.replaceAll("+")
-      (allMatchedCharsPluses.toSet filterNot (_ == '+')).mkString(", ")
+      (allMatchedCharsPluses.toSet filterNot (_ == '+')).mkString("  ")
```
Comment (tmikus) [inline src/main/scala/uk/gov/hmrc/ct/box/ValidatableBox.scala]: And here.

File: src/test/scala/uk/gov/hmrc/ct/accounts/frs10x/AccountsFreeTextValidationFixture.scala (hunk 1 above)
Comment (mario-paniccia) [inline src/test/scala/uk/gov/hmrc/ct/accounts/frs10x/AccountsFreeTextValidationFixture.scala]: yes that's part of the requirements. Kaye asked for a double space between illegal characters

**Pull request merged**